tests/infer/test_elbo_mapdata.py

```diff
@@ -27,6 +27,8 @@ def test_elbo_mapdata(batch_size, map_type):
     lam = torch.tensor([6.0, 4.0])
     data = []
     sum_data = torch.zeros(2)
+    params = {("iplate", 8): (100, 0.018), ("iplate", None): (100, 0.013),
```

Comment:
nit: can you replace these sorts of thing with `mark.parameterize' usage?

see e.g. [here](https://github.com/pyro-ppl/pyro/blob/1888efc05199bcb9319d5d4869d417988a4104d8/tests/contrib/autoguide/test_mean_field_entropy.py#L26)

Reply:
done